### DISCUSSION TITLE
Target Class [AuthController] Does not Exist

### DISCUSSION THREAD

**Discussion by matiusnugroho** [enlightn/enlightn#102] [Q&A]
I Have AuthController in my web.php route but enlightn said it does not exist
here is my route
```
use App\Http\Controllers\AuthController;
Route::get('/login',[AuthController::class,'showLoginForm'])->name('login');
Route::post('/login',[AuthController::class,'login'])->name('auth.login');
Route::get('/logout',[AuthController::class,'logout'])->name('auth.logout');
```
what is going wrong and how to fix it?

**Comment by perisicnikola37** [comment]
1. check for the namespace of that controller because if it is in some folder you need to specify it like this:
`use App\Http\Controllers\Folder\AuthController;`
2. run the command to clear the cache:
`php artisan optimize`
maybe your routes are cached...

I see that you want login routes..
Why not use this?
`
Route::controller(LoginController::class)->group(function() {
Route::get('/login', 'showLoginForm')->name('login');
Route::post('/login', 'login');
Route::get('/logout', 'logout');
});
`
